src/custom-elements/src/components/spinner/spinner.scss

```diff
@@ -0,0 +1,4 @@
+:host(chi-spinner) {
+  @import '../../global/styles/common';
+  @import 'components/spinner/spinner';
+}
```

Comment:
Please, add default display and backdrop scss: should be something like this:
```
:host(chi-spinner) {
	  @import '../../global/styles/common';
	  @import 'components/spinner/spinner';
	  @import 'components/backdrop/backdrop';
    display: inline;
    &[backdrop] {
      display: block;
    }
	}
```
Remember, as far as I know, most custom-element display property should match the first element inside it. This will prevent element misplacements.